release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -80,6 +80,11 @@ This release introduces the following updates to the *Developer* perspective of
 [id="ocp-4-16-openshift-cli"]
 === OpenShift CLI (oc)
 
+[id="ocp-4-16-oc-adm-upgrade-status-tp"]
+==== Introducing `oc adm upgrade status` command (Tech Preview)
+
+Previously, customers could retrieve information about the status of their cluster update from the web console and from the CLI. In addition to its primary focus, the `oc adm upgrade` command provided some limited information about the upgrade status. With this release, there is now a tech preview `oc adm upgrade status` command which decouples the status information from `oc adm upgrade` and compliments the UI by providing customers with specific information regarding their cluster update, including the status of the control plane and worker node updates.
```

Comment:
Can customers still retrieve some update info through the console and CLI without the upgrade status command? That first sentence starting with "Previously" makes it seem like that functionality is going away.

If that functionality isn't going away, I think you could either remove those first two sentences entirely (since the focus is on new features anyways), or maybe focus that initial description on how users only got limited info from `oc adm upgrade`, like this:
```suggestion
Previously, customers could run the `oc adm upgrade` command to retrieve limited information about the cluster update status. With this release, there is now a tech preview `oc adm upgrade status` command which decouples the status information from `oc adm upgrade` and compliments the UI by providing customers with specific information regarding their cluster update, including the status of the control plane and worker node updates.
```

Reply:
I think may be we can say "We extended oc adm upgrade, previous xxxx, and now customer can run `oc adm upgrade` for simple info and run  `oc adm upgrade status` for detail info".

Reply:
I think we should focus on the functionality of `oc adm upgrade status` here, but I can touch on the more specific capabilities of `oc adm upgrade` and finer details of `oc adm upgrade status` in the upgrade section :)